recipes/slncky/run_test.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+set -x
+set -e
+
+echo "PATH is $PATH"
+# The annotations directory needs to be uploaded by the user.
+# It is too big to be uploaded automatically.
+# test -d $PREFIX/annotations
+slncky --help > /dev/null 2>&1
```

Comment:
can you this simply put in the meta.yaml file in a test section?

Reply:
Yes. I will do that.